README.md

```diff
@@ -1,7 +1,7 @@
 [![Gem Version](https://badge.fury.io/rb/intelipost.svg)](http://badge.fury.io/rb/intelipost)
-[![Build Status](https://travis-ci.org/natuelabs/intelipost.svg?branch=master)](https://travis-ci.org/natuelabs/intelipost)
-[![Code Climate](https://codeclimate.com/github/natuelabs/intelipost/badges/gpa.svg)](https://codeclimate.com/github/natuelabs/intelipost)
-[![Test Coverage](https://codeclimate.com/github/natuelabs/intelipost/badges/coverage.svg)](https://codeclimate.com/github/natuelabs/intelipost/coverage)
+[![Build Status](https://travis-ci.org/Gaveteiro/intelipost.svg?branch=master)](https://travis-ci.org/Gaveteiro/intelipost)
```

Comment:
@rwillians Sorry, my bad. I didn't knew we already had a PR opened here. Will fix.